nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -293,6 +293,10 @@ SECRET_KEY = os.getenv("NAUTOBOT_SECRET_KEY", "{{ secret_key }}")
 # CELERY_TASK_SOFT_TIME_LIMIT = int(os.getenv("NAUTOBOT_CELERY_TASK_SOFT_TIME_LIMIT", str(5 * 60)))
 # CELERY_TASK_TIME_LIMIT = int(os.getenv("NAUTOBOT_CELERY_TASK_TIME_LIMIT", str(10 * 60)))
 
+# Port for prometheus metric HTTP server running on the celery worker
+# Set this to 0 to disable it.
```

Comment:
Please duplicate the very helpful comments from `settings.py` here so that they're more visible to a user setting up a new config file.